### ISSUE TITLE
No way to get mining reward

### ISSUE THREAD

**Issue by yma-het** [abeychain/go-abey#9]
Currently coinbase address is just ignored in miner: https://github.com/abeychain/go-abey/blob/552bd86c79e3927f38be4a24854a88d758076b5f/miner/miner.go#L355
Even if we pass correct address as it is mentioned in CLI doc: https://github.com/abeychain/go-abey/wiki/Command-Line-Options
mined fruit will be dropped here: https://github.com/abeychain/go-abey/blob/552bd86c79e3927f38be4a24854a88d758076b5f/core/snailchain/snail_pool.go#L266-L271
Ok, even if we bypass this restriction, when we attempt to insert mined fruit into block, it will be rejected: https://github.com/abeychain/go-abey/blob/552bd86c79e3927f38be4a24854a88d758076b5f/core/snailchain/block_validator.go#L184-L193

If we liook into mined blocks: https://scan.abeychain.com/snail only one address is minig and it is 0xD9DeC020337DAeB794936Bc0A6Ead8E343cb9B6c
If we will try to look at mining reward for ayny block we will see no fields BlockMinerReward or FruitMinerReward:
curl 'https://rpc.abeychain.com/' --header 'Content-Type: application/json' --data-raw '{
"jsonrpc":"2.0",
"method":"abey_getSnailRewardContent",
"params":["0x9b5f"],
"id":1
}'
only PoS rewards (committeeReward).

**Comment by gawainchoo** [comment]
> .NumberU64() > 233 &&

Right, by reading the codes , we could find that only one addr is available to accept rewards， if we wanna mine ABEY TOKEN by mining machines. like , what you said,0xD9DeC020337DAeB794936Bc0A6Ead8E343cb9B6c.
by stepping  into this project, seem that only one mine pool works in abeychain .other participants get objected. Therefore, we need to purchase hash rate to mine ABEY TOKEN from the project party, or be an ABEY HOLDER to join POS mine.
